app/pkg/dbx/migrate.go

```diff
@@ -140,3 +143,31 @@ func getLastMigration() (int, error) {
 
 	return int(lastVersion.Int64), nil
 }
+
+func getPendingMigrations(versions []int) ([]int, error) {
+	pendingMigrations := make([]int, 0)
+	versionStr := strconv.Itoa(versions[0])
+
+	for _, version := range versions {
+		versionStr = versionStr + "," + strconv.Itoa(version)
+	}
```

Comment:
I think the first version number is duplicated here. While not a big deal because of how it's used later, I think this should do the trick:
```suggestion
	versionStr := ""

	for _, version := range versions {
		versionStr = versionStr + "," + strconv.Itoa(version)
	}
	versionStr = versionStr[1:]
```
(I did not tested this code and the last time I coded in Go is quite a long ago so I might be wrong, but it looks OK)